### PR TITLE
plugin tutorial, some reordering of folders etc... new pull request

### PR DESCRIPTION
**Pull request by theweekendgeek**
added some notes directives and fixed indentation in one line

### CODE REVIEW

File: dev/plugin_dev.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
+==================
+Plugin Development
+==================
+
+A working plugin for aiPhilos is easy to develop and requires only three steps to actually get started.
```
Comment (mhilbrunner) [inline dev/plugin_dev.rst]: It's easy to develop a working plugin for aiPhilos
(use active voice)

File: dev/plugin_dev.rst [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+This guide assumes that you have at least created an aiPhilos account and that you have created an user so you can test your plugin against our api.
```
Comment (mhilbrunner) [inline dev/plugin_dev.rst]: Capitalize API

File: dev/plugin_dev.rst [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+================================
+
+ You can create a database by synching your items via a **POST-Request** against our batch-endpoint
+
+ the url for the batch-endpoint is: 
+
+ .. literalinclude:: ../shared/endpoints/POST_batch.txt
+
+
+ Because of the time being, aiPhilos is only available in german, you will need to **replace {langauge} with 'de' (without the quotes) and {db} with the database name of your choice**.
```
Comment (mhilbrunner) [inline dev/plugin_dev.rst]: For the time being

File: getting_started/index.rst [hunk 4]
```diff
@@ -1,12 +0,0 @@
-aiPhilos Integration
-====================
-
-aiPhilos is usually utilized via the use of small applications, usually called plugins or addons.
```
Comment (mhilbrunner) [inline getting_started/index.rst]: remove "small", comma

File: getting_started/index.rst [hunk 5]
```diff
@@ -1,12 +0,0 @@
...
-These small applications are then integrated into third party solutions like Shopware and enable them to leverage the capability of aiPhilos semantic search.
```
Comment (mhilbrunner) [inline getting_started/index.rst]: remove "small"

File: getting_started/integration.rst [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Integration
+===========
+
+The semantic search of aiPhilos is utilized via the use of small applications, usually called plugins or addons.
+These small applications are then integrated into third party solutions (e. g. shop systems like Shopware) and enable them to leverage the capability of aiPhilos.
```
Comment (mhilbrunner) [inline getting_started/integration.rst]: capabilities

File: dev/plugin_dev.rst (hunk 3 above)
Comment (schwaen) [reply dev/plugin_dev.rst]: de-de instead de

**Pull request merged**